mod/wikirate/set/all/contributions.rb

```diff
@@ -21,11 +22,11 @@ def update_contribution_count
       0
     end
     
-  if respond_to? :indirect_contributer_search_args
-    indirect_contributer = indirect_contributer_search_args.inject([]) do |cards, search_args|
+  if respond_to? :indirect_contributor_search_args
+    indirect_contributor = indirect_contributor_search_args.inject([]) do |cards, search_args|
       cards += Card.search(search_args)
     end
-    new_contr_count += indirect_contributer.inject(0) do |res,c_card| 
+    new_contr_count += indirect_contributor.inject(0) do |res,c_card|
       res += if c_card.respond_to?(:contribution_count)
```

Comment:
this was the main problem: 'contributer' instead of 'contributor'. I wonder how the tests could pass before. The typo is there since October.